--- CMakeLists.txt~	2018-10-17 10:49:25.000000000 +0300
+++ CMakeLists.txt	2018-10-17 11:23:26.351449716 +0300
@@ -622,7 +622,7 @@
     "${CMAKE_CURRENT_BINARY_DIR}/FreeGLUT/FreeGLUTConfig.cmake"
     @ONLY
 )
-SET(ConfigPackageLocation lib/cmake/FreeGLUT)
+SET(ConfigPackageLocation ${CMAKE_INSTALL_LIBDIR}/cmake/FreeGLUT)
 INSTALL(EXPORT FreeGLUTTargets
     FILE FreeGLUTTargets.cmake
     NAMESPACE FreeGLUT::
